core/templates/domain/story/story-contents-object.model.ts

```diff
@@ -244,6 +244,52 @@ export class StoryContents {
     }
   }
 
+  setNodeStatus(nodeId: string, status: string): void {
+    var index = this.getNodeIndex(nodeId);
```

Comment:
```suggestion
    let index = this.getNodeIndex(nodeId);
```
Avoid using `var`, use `let`.

Reply:
Done